### DISCUSSION TITLE
Is `TracePoint.new(:raise)` not supported in truffleruby?

### DISCUSSION THREAD

**Discussion by stdedos** [oracle/truffleruby#3679] [Q&A] [answered]
I am trying to build a quick pipeline for https://github.com/ConradIrwin/interception (https://github.com/stdedos/interception/commit/4ff277cf419ce1bdb4af7a4fcd1f897d88ee3222), but it seems that "no truffleruby variant" supports `TracePoint.new(:raise)`?

https://github.com/stdedos/interception/actions/runs/11141771488/job/30963350904#step:5:19

I am not very involved with ruby - but you do claim:

> https://www.graalvm.org/ruby/
> TruffleRuby aims to be fully compatible with the standard Ruby runtime (MRI/CRuby)

So I thought it is not far-fetched to at least ask for some quick tips 🙏

**Comment by eregon** [comment, marked as answer]
I moved it to an issue and will reply there: https://github.com/oracle/truffleruby/issues/3680
